sdk/python/feast/feature_set.py

```diff
@@ -230,6 +233,23 @@ def max_age(self, max_age):
         """
         self._max_age = max_age
 
+    @property
+    def label(self):
```

Comment:
Label isnt a property of a feature set, its a property of a feature.

Reply:
My bad, Thanks for clarifying

Hence, this is current to do Checklist:

- Add Mac development set-up guide using homebrew
- Create the `labels` column on Field Class (This will affect the Feature and Entity)  
- Create a gateway to Create, List, and Delete to the labels Column in `Feature` class using python SDK Client
- Add the test with labels Column empty and labels Column marked in Feature class
- Update the integration test tree
- Update the documentation on how to use the labels from Feature function

Notes, Datatype for each of the layers
```
Python: dict()
Java: Map<String, String>
Table: Varchar(255): with key-value object store in JSON
```

Reply:
`Varchar(255): with key-value object store in JSON`

Why are you limiting it to that small of a size? I also think you should maybe consider comma or tab separation here and using a Set (in the Java model)

Reply:
Sure @woop , now I am using 'TEXT' for a result similar with Varchar(Max), thanks for the input